lib/mapping.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
   ircToSlack: function(originalMessage) {
     var modifiedMessage = originalMessage;
     _.each(userMap, function(val, key, list) {
-      var re = new RegExp("@?" + key, 'gi');
+      var re = new RegExp("@?" + key + '\s', 'gi');
```

Comment:
Yeah, it does... I've added another test for that case.  Now I'm just trying to get it to pass.  I'll push it up as soon as I do get it working.